### PR TITLE
Hamburger in chat

### PR DESCRIPTION
**Pull request by tharain**
- added nojs script to be called when one turn of JavaScript
- added hamburger in chatvue

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ module.exports = {
     // forbid trailing commas
     'comma-dangle': ['error', 'never'],
     // allow debugger during development
-    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0
+    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
+	"linebreak-style": 0
```
Comment (KnightNiwrem) [inline .eslintrc.js]: Identation problem

File: src/views/ChatView.vue [hunk 2]
```diff
@@ -69,15 +86,53 @@ export default {
...
+    Alert
   },
   methods: {
     exitChat() {
       this.$router.push({ path: '/' });
+    },
+    anom(anonymous) {
+      let alert = null;
+      if (anonymous) {
+        document.getElementsByClassName('q-toggle-handle')[0].style = 'background-color: rgb(27, 188, 155); border: none;';
```
Comment (KnightNiwrem) [inline src/views/ChatView.vue]: I don't recommend this. Consider, at least, toggling a classname on the element, where the existence of that classname modifies its styling.

**Review by KnightNiwrem — DISMISSED** [review]

File: src/views/ChatView.vue [hunk 3]
```diff
@@ -69,15 +86,53 @@ export default {
...
+        });
+      }
+      window.setTimeout(() => {
+        alert.dismiss();
+      }, 10);
     }
   },
   data() {
     return {
+      anonymous: true,
```
Comment (ngsikai) [inline src/views/ChatView.vue:132]: I forgot, did we decide on having the anonymous setting appear on an app level or room level? I imagine we need to have an endpoint for this. Or localstorage lol @tharain @KnightNiwrem

File: src/views/ChatView.vue [hunk 4]
```diff
@@ -166,4 +221,25 @@ export default {
 <style scoped lang="stylus">
 .message-input
   margin: 0
+.q-table
+  border: 10px solid rgb(52,73,94)
+.q-list
```
Comment (ngsikai) [inline src/views/ChatView.vue]: be careful with styles like this that might impact someone else using a list the next time. Same potential issue with `button` and `.q-popover`

File: src/views/ChatView.vue (hunk 2 above)
Comment (ngsikai) [reply src/views/ChatView.vue]: Agreed

File: src/views/ChatView.vue [hunk 5]
```diff
@@ -69,15 +86,53 @@ export default {
...
+    QItem,
+    QItemMain,
+    QOptionGroup,
+    Alert
   },
   methods: {
     exitChat() {
       this.$router.push({ path: '/' });
+    },
+    anom(anonymous) {
```
Comment (ngsikai) [inline src/views/ChatView.vue]: toggleAnom or setAnom instead of anom?

**Review by ngsikai — DISMISSED** [review]

File: src/views/ChatView.vue (hunk 2 above)
Comment (tharain) [reply src/views/ChatView.vue]: I don't get this sorry. Can explain further?

File: .eslintrc.js (hunk 1 above)
Comment (tharain) [reply .eslintrc.js]: ok fixed thanks :D

File: src/views/ChatView.vue (hunk 5 above)
Comment (tharain) [reply src/views/ChatView.vue]: ok fixed thanks :D

File: src/views/ChatView.vue (hunk 3 above)
Comment (tharain) [reply src/views/ChatView.vue:132]: ops I'm just following sj drawing on PowerPoint. Ya can change if needed.

Comment (KnightNiwrem) [reply src/views/ChatView.vue:132]: Let's put the details of this off for now. This doesn't seem to be a huge implementation that we can't change later. So, accept.

**Review by KnightNiwrem — APPROVED** [review]

**Pull request merged**